api/evaluation/v1alpha1/rag_types.go

```diff
@@ -49,12 +50,26 @@ type RAGSpec struct {
 
 	// Report defines the evaluation report configurations
 	Report Report `json:"report,omitempty"`
+
+	// Storage storage must be provided and data needs to be saved throughout the evaluation phase.
+	Storage *corev1.PersistentVolumeClaimSpec `json:"storage"`
+
+	// ServiceAccountName define the user when the job is run
+	// +kubebuilder:default=default
+	ServiceAccountName string `json:"serviceAccountName,omitempty"`
```

Comment:
@0xff-dev can you provide a sample file under config/samples to create a service account which has the required permissions to run this eval.

Reply:
And please add the `CommonSpec` to this RAGSpec

Reply:
This `ServiceAccountName` is defined for the job, and currently in the first two phases, it will ask for permissions on `datasource`, `versioneddataset`, and `configmap`, `Application`, `RAG`, and the last two phases are not clear about the permissions.

Reply:
ok